### PR TITLE
Tickets/damd 140

### PR DESCRIPTION
**Pull request by CraigLoomis**

### CODE REVIEW

File: python/pfs/datamodel/pfsConfig.py [hunk 1]
```diff
@@ -415,6 +431,11 @@ def _readHeader(cls, header, kwargs=None):
         elif "pfsDesignId" not in kwargs:
             raise RuntimeError("Unable to determine pfsDesignId")
 
+        # Load design variant cards, added for DAMD-140. If none are set, declare the design
+        # as having none.
```
Comment (hassanxp) [inline python/pfs/datamodel/pfsConfig.py:435]: In comment, change 'none' to 'no variant', as the new instance will have variant cards set to ``0``, correct?

**Review by hassanxp — APPROVED** [review]
Probably need an extra test for ``calculate_pfsDesignId()``, to check that the SHA does not change if ``variant`` is not specified, or ``variant==0``, and that the SHA changes with different non-zero values of ``variant``.

File: python/pfs/datamodel/pfsConfig.py (hunk 1 above)
Comment (CraigLoomis) [reply python/pfs/datamodel/pfsConfig.py:435]: There is already a test for two variants having different hashes.

I thought that differentiating a new variant=0 from an older no variant design was slight obnoxious: we would need to save a static hash for an older design to compare against.

Comment (CraigLoomis) [reply python/pfs/datamodel/pfsConfig.py:435]: Sure on clarifying that "none" vs "None"

**Pull request merged**